src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,18 @@ public class RegistrationServiceImpl implements RegistrationService {
     private static final int MIN_LOGIN_LENGTH = 6;
     private static final int MIN_PASSWORD_LENGTH = 6;
     private static final int MIN_AGE = 18;
+    private static final String EXCEPTION_MESSAGE_NULL_USER =
+            "User shouldn't be null.";
+    private static final String EXCEPTION_MESSAGE_NULL_FIELDS =
+            "All fields should be filled.";
+    private static final String EXCEPTION_MESSAGE_MIN_LOGIN =
+            "The login should be at least 6 characters.";
```

Comment:
great job! I meant that you don't move whole message to constants, but use text including constant field

"The login should be at least + MIN_LOGIN_LENGTH + characters."